fe/src/static/sass/base.scss

```diff
@@ -1,7 +1,53 @@
-$font-stack:    Helvetica, sans-serif;
+/***********************************************
+			DEFAULT STYLES
+***********************************************/
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
+$font-stack: 'Montserrat', Helvetica, sans-serif;
+
+$dark-black: rgb(15, 15, 15);
+$light-black: rgb(22, 22, 22);
+$green: rgb(24, 177, 62);
 $primary-color: #333;
 
 body {
-  font: 100% $font-stack;
-  color: $primary-color;
+	margin: 0;
```

Comment:
Please adhere to project convention for indentations.
Unit of indentation is 2 spaces not a tab character.
To map tab to 2 spaces on sublime please goto 
Sublime Text -> Preferences -> User Settings
and type the following:

"tab_size": 2,
"translate_tabs_to_spaces": true,